environment/scripts/setup_data.py

```diff
@@ -275,5 +587,18 @@ def download_multiplier_recount2_model(**kwargs):
 
     method_args = vars(args)
 
-    for method_name, method in AVAILABLE_ACTIONS[args.mode]:
+    methods_to_run = {}
+
+    if args.action is not None:
+        if args.action not in AVAILABLE_ACTIONS["full"]:
+            import sys
```

Comment:
Just checking whether it's more efficient to load this here or just move it to the top!

Reply:
It's the only place where I use the `sys` module, so I think it makes sense to leave it here. Thanks for checking!